pkgs/development/libraries/openmpi/default.nix

```diff
@@ -29,6 +33,14 @@ in stdenv.mkDerivation rec {
     sha256 = "0ms0zvyxyy3pnx9qwib6zaljyp2b3ixny64xvq3czv3jpr8zf2wh";
   };
 
+  # TODO: Remove these, when a new release gets out
+  # https://github.com/open-mpi/ompi/commit/526775dfd7ad75c308532784de4fb3ffed25458f.patch
+  # https://github.com/open-mpi/ompi/commit/a3026c016a6a8be379f62585b6ddc070175c8106.patch
+  patches = [
+    ./ucx-1.8.patch
```

Comment:
There is only UCX 1.7?

Reply:
yes, but the patch is in preparation for the 1.8 release and they only later realized that 1.7 was broken as well.

Reply:
There should not be unnecessary patches. If it becomes relevant with the next release then it should be added. Consider also to use `fetchPatch` for patches that can be fetched online.